dev/Generated/TabView.properties.cpp

```diff
@@ -10,7 +10,6 @@ CppWinRTActivatableClassWithDPFactory(TabView)
 
 GlobalDependencyProperty TabViewProperties::s_AddButtonCommandProperty{ nullptr };
 GlobalDependencyProperty TabViewProperties::s_AddButtonCommandParameterProperty{ nullptr };
-GlobalDependencyProperty TabViewProperties::s_CanCloseTabsProperty{ nullptr };
```

Comment:
I'm curious about this change: was the idea in API review to just require devs to always set this on the tabs themselves, rather than globally on the TabView like this?

Reply:
Yes, that was the decision.

Reply:
Makes sense. :)  Thanks.